config/webpack/alias.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
       neetoui: "@bigbinary/neetoui",
       neetoicons: "@bigbinary/neeto-icons",
       utils: "src/utils",
+      dayJS: "src/lib/dayjs",
```

Comment:
What's the point of this alias?

Reply:
It was added as per [this](https://github.com/navaneethsdk/wheel/pull/17#discussion_r979204569) request.

Reply:
@navaneethsdk Add an alias for `lib` folder, so that once we have more 3rd party extensions under `lib` folder we could easily reference it with an alias rather than using the relative path. 

Don't add an alias for dayjs. What's the advantage that you saw by adding an alias for `dayjs` when we are only importing it at one place in the whole codebase? If you didn't understand the point of a review comment or what it's suggesting, then make sure to clarify the same with the reviewer before jumping in and making the change.

Reply:
Oh, I am sorry Yedhin. I thought this was what you meant (that is why I did not raise doubt). I will rectify this at once and make sure such errors are not repeated.

Reply:
I have fixed this in #28